release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,832 @@
+workloads.
+
+You can expand persistent volume claims online from {product-title} for glusterFS.
+
+. Create a storage class with `allowVolumeExpansion=true`.
+. The PVC uses the storage class and submits a claim.
+. The PVC specifies a new increased size.
+. The underlying PV is resized.
+
+See
+xref:../dev_guide/expanding_persistent_volumes.adoc#expanding_persistent_volumes[Expanding
+Persistent Volumes] for more information.
+
+[[ocp-311-tenant-driven-storage-snapshotting]]
+==== Tenant-driven Storage Snapshotting (Technology Preview)
+
+Tenant-driven storage snapshotting is currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+Tenants now have the ability to leverage the underlying storage technology
+backing the persistent volume (PV) assigned to them to make a snapshot of their
+application data. Tenants can also now restore a given snapshot from the past to
+their current application.
+
+An external provisioner is used to access the EBS, GCE pDisk, and HostPath. This
+Technology Preview feature has tested EBS and HostPath. The tenant must stop the
+pods and start them manually.
+
+. The administrator runs an external provisioner for the cluster. These are images
+from the Red hat Container Catalog.
+
+. The tenant made a PVC and owns a PV from one of the supported storage
+solutions.The administrator must create a new `StorageClass` in the cluster with:
++
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: snapshot-promoter
+provisioner: volumesnapshot.external-storage.k8s.io/snapshot-promoter
+----
+
+. The tenant can create a snapshot of a PVC named `gce-pvc` and the resulting
+snapshot will be called `snapshot-demo`.
++
+----
+$ oc create -f snapshot.yaml
+
+apiVersion: volumesnapshot.external-storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot-demo
+  namespace: myns
+spec:
+  persistentVolumeClaimName: gce-pvc
+----
+
+. Now, they can restore their pod to that snapshot.
++
+----
+$ oc create -f restore.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: snapshot-pv-provisioning-demo
+  annotations:
+    snapshot.alpha.kubernetes.io/snapshot: snapshot-demo
+spec:
+  storageClassName: snapshot-promoter
+----
+
+[[ocp-311-scale]]
+=== Scale
+
+[[ocp-311-scale-cluster-limits]]
+==== Cluster Limits
+
+Updated guidance around
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[Cluster
+Limits] for {product-title} 3.11 is now available.
+
+[[ocp-311-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-311-prometheus]]
+==== Prometheus Cluster Monitoring
+
+Prometheus cluster monitoring is now fully supported in {product-title}.
+
+You can deploy Prometheus on an {product-title} cluster, collect Kubernetes and
+infrastructure metrics, and get alerts. You can see and query metrics and alerts
+on the Prometheus web dashboard. Alternatively, you can bring your own Grafana
```

Comment:
There is a pre-configured Grafana out of the box. Bring your own Grafana is currently not supported in this setup.

@sichvoge can you please clarify this with the other cluster-monitoring entry in here?